src/main/resources/messages.properties

```
@@ -407,6 +407,14 @@ next-steps-email.body=<html><body><p><b>Time to Hear Back:</b><br>In the next 7-
 #
 phone-number-nudge.title=Phone number nudge
 phone-number-nudge.header=Are you sure you want to leave your phone number blank?
-phone-number-nudge.subheader=A caseworker may need to contact you by phone about your application. If you don't have a phone number, you can enter a friend or family member's phone number instead.
+phone-number-nudge.subheader=A caseworker may need to contact you by phone about your application. If you don't have a phone number, you can enter a friend or family member's phone number instead. 
```

Small nick pick: remove the white space at the end of the line.

```suggestion
phone-number-nudge.subheader=A caseworker may need to contact you by phone about your application. If you don't have a phone number, you can enter a friend or family member's phone number instead.
```

---

@coltborg I'm pushing code to fix this.